docs/scenario_editor/scenarios.md

```diff
@@ -5,7 +5,7 @@ controlling various different aspects of AI vehicle behavior. There are two type
 
```

Comment:
Need to change the above paragraph:
"The monoDrive Scenario Editor allows users to create their own scenarios by controlling various different aspects of AI vehicle behavior. There are two types of files that can be created: the trajectory file for Replay Mode and the scenario file for Closed loop mode."

Try something like this (feel free to change it):
"The monoDrive Scenario Editor allows users to create their own scenarios by controlling various different aspects of AI vehicle behavior, these scenarios are used when running the simulator or editor on closed loop mode"